### PR TITLE
Update to support 0.7 API

### PR DESCRIPTION
**Pull request by cedk**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,33 +1,23 @@
...
-    username: $$DOCKER_USER
-    password: $$DOCKER_PASS
-    email: $$DOCKER_EMAIL
-    repo: plugins/drone-hg
-    tag: develop
-    when:
-      branch: develop
+      event: push
 
 plugin:
```
Comment (tboerger) [inline .drone.yml]: You can drop this section, it's not used anymore.

File: README.md [hunk 2]
```diff
@@ -6,162 +6,44 @@
...
-        "root": "/drone/src",
-        "path": "/drone/src/bitbucket.org/drone/drone"
-    }
-}
-EOF
-```
+## Usage
 
-### Clone a pull request
+Clone a commit:
```
Comment (tboerger) [inline README.md:40]: Can you add more samples like https://github.com/drone-plugins/drone-git#usage?

**Comment by tboerger** [comment]
Beside my 2 NITPICK comments it LGTM.

File: README.md (hunk 2 above)
Comment (cedk) [reply README.md:40]: But there are no other event pull request or tag like in Git.

**Review by tboerger — APPROVED** [review]

**Pull request merged**